CMakeLists.txt

```diff
@@ -16,12 +16,12 @@ include(ExternalProject)
 ## OPTIONS
 ##
 option(JSON_BuildTests "Build the unit tests when BUILD_TESTING is enabled." ON)
+option(JSON_MultipleHeaders "Use non-amalgamated version of the library." OFF)
```

Comment:
Why would anyone want this?

Reply:
To include the `json_fwd.hpp` file, or the `adl_serializer.hpp`. That's something I want to use at work, I like to only include what I need.

Reply:
Ok.